sql/hive/src/test/scala/org/apache/spark/sql/hive/LogicalPlanToSQLSuite.scala

```diff
@@ -145,6 +145,16 @@ class LogicalPlanToSQLSuite extends SQLBuilderTest with SQLTestUtils {
     checkHiveQl("SELECT COUNT(DISTINCT id) FROM t0")
   }
 
+  test("TABLESAMPLE") {
+    checkHiveQl("SELECT * FROM t0 TABLESAMPLE(100 PERCENT) s")
```

Comment:
Testing tables created in this test suite are all Parquet tables, which are by default converted from `MetastoreRelation` to `ParquetRelation` wrapped with a `Subquery`. We should probably rename such tables, e.g., `t0` to `parquet_t0`, and add real text format `MetastoreRelation` testing tables via explicit HiveQL DDLs.